docs/src/main/resources/examples/README.compactionStrategy

```diff
@@ -0,0 +1,60 @@
+Title: Apache Accumulo Customizing the Compaction Strategy 
+Notice:    Licensed to the Apache Software Foundation (ASF) under one
+           or more contributor license agreements.  See the NOTICE file
+           distributed with this work for additional information
+           regarding copyright ownership.  The ASF licenses this file
+           to you under the Apache License, Version 2.0 (the
+           "License"); you may not use this file except in compliance
+           with the License.  You may obtain a copy of the License at
+           .
+             http://www.apache.org/licenses/LICENSE-2.0
+           .
+           Unless required by applicable law or agreed to in writing,
+           software distributed under the License is distributed on an
+           "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+           KIND, either express or implied.  See the License for the
+           specific language governing permissions and limitations
+           under the License.
+
+This tutorial uses the following Java classes, which can be found in org.apache.accumulo.tserver.compaction: 
+
+ * DefaultCompactionStrategy.java - determines which files to compact based on table.compaction.major.ratio and table.file.max
+ * EverythingCompactionStrategy.java - compacts all files
+ * SizeLimitCompactionStrategy.java - compacts files no bigger than table.majc.compaction.strategy.opts.sizeLimit
+ * TwoTierCompactionStrategy.java - uses default compression for smaller files and table.majc.compaction.strategy.opts.file.large.compress.type for larger files
+
+This is an example of how to configure a compaction strategy. By default Accumulo will always use the DefaultCompactionStrategy, unless 
+these steps are taken to change the configuration.  Use the strategy and settings that best fits your Accumulo setup.
+
+The command below sets the compression for smaller files and minor compactions.
```

Comment:
The 4 config commands below are at the system level, so the settings would apply to all tables.  I think an example should show doing this at the table level.  Could create the table and then set the four configuration params for the table.